### ISSUE TITLE
[Access] Test pebble execution data db on testnet

### ISSUE THREAD

**Issue by peterargue** [onflow/flow-go#6357]
Using an observer on testnet, test running execution sync using a pebble db datastore.

Monitor logs and metrics for any errors or halts while syncing, or any other signs of performance issues.

You can use the configs from localnet to setup a local version of grafana to view metrics if that's useful

**Comment by peterargue** [comment]
@UlyanaAndrukhiv can you add details about how to set this up

**Comment by UlyanaAndrukhiv** [comment]
The Pebble execution data DB setup requires the following flags:
```
--execution-data-db=pebble
--execution-data-dir=<path_to_pebble_db>
--execution-data-sync-enabled=true
```

**Comment by peterargue** [comment]
I think the way to gets the data in the db is correct is to do something like this:

1. Get the node's root block using the `NodeVersionInfo` endpoint
2. Setup a stream of execution data starting from the node's root block.
3. Use [`execution_data.CalculateID`](https://github.com/onflow/flow-go/blob/master/module/executiondatasync/execution_data/util.go#L12-L21) to calculate the root ID for the `BlockExecutionData` object.
4. Use `GetExecutionResultByBlockID` to get the execution result for the block, and compare the ID calculated in 3 with the `ExecutionDataID` field. If they match, then the data is identical

**Comment by peterargue** [comment]
as discussed in standup, please test using `BlockExecutionData` instead of events.

**Comment by peterargue** [comment]
I don't think this is going to work using json. you will need to use grpc to get the binary data from the GetExecutionDataByBlockID (or streaming version) with CCF encoded events, then use the code you have above.

`executionDataID` won't equal the block ID, it should match the `ExecutionDataID` from the `ExecutionResult` for the block. You can use `GetExecutionResultByBlockID` to get the correct result.
